iree_tests/README.md

```diff
@@ -88,7 +88,7 @@ To use local versions of `iree-compile` and `iree-run-module`, put them on your
 `$PATH` ahead of your `.venv/Scripts` directory:
 
 ```bash
-$ export PATH=path/to/iree-build;$PATH
+$ export PATH=path/to/iree-build:$PATH
 ```
```

Comment:
Ah, I wrote this for Windows, where the path separator is `;`. It is indeed `:` on Linux. Fine to use Linux syntax here, as that matches the other docs (venv setup is a bit different on Windows too, for example). In more expressive markdown / doc formats I'll use content tabs like those here: https://iree.dev/reference/bindings/python/#prerequisites

Linux selected: 
![image](https://github.com/user-attachments/assets/0d2675d4-d8a1-4a2a-b831-293f41f3e8d0)

Windows selected:
![image](https://github.com/user-attachments/assets/a450d4d3-06ae-4c95-970d-cb65f60ac52b)